### PR TITLE
06 reducer docs

### LINKED ISSUE #6: Document reducer functions
As a new developer, I want an easy way to find out what the different reducer functions do and how to use them, so that I can get up-to-speed more quickly.

Add [jsdoc](https://jsdoc.app/)-style docblocks to all of the functions in `client/reducers/*.js` (not including tests).

# Acceptance Criteria
* All functions in `client/reducers/*.js` (not including tests) have a docblock that includes:
    * The function's purpose.
    * The type and description of the function's parameters.
    * The type and description of the function's return value.

### PR DESCRIPTION
**Pull request by emma-r-slight**

### CODE REVIEW

**Review by stevie-jack-wilder — APPROVED** [review]
Looks great!

File: client/reducers/createUser.js [hunk 1]
```diff
@@ -1,10 +1,25 @@
...
+
+/** @type {userState}  */
 const InitialUserInfo = {
   id: '',
   username: ''
 }
 
+/**
+ * CreateUser reducer updates userState in Store
+ * @param {userState} state - initial  userState object
```
Comment (todofixthis) [inline client/reducers/createUser.js]:
```suggestion
 * @param {userState} state - initial userState object
```

File: client/reducers/createUser.js [hunk 2]
```diff
@@ -1,10 +1,25 @@
...
+ * @param {string} action  - add user to state
+ * @returns {object} - updated state object
```
Comment (todofixthis) [inline client/reducers/createUser.js]:
```suggestion
 * @returns {userState} - updated state object
```

File: client/reducers/setEvents.js [hunk 3]
```diff
@@ -1,5 +1,12 @@
 import { SET_EVENTS } from '../actions'
 
+/**
+ * setEvents reducer the adds the setEvents object to the store
+ * @param {object[]} state - current array of events in state
+ * @param {string} action - action to update events array
+ * @returns {array}  updated state
```
Comment (todofixthis) [inline client/reducers/setEvents.js]:
```suggestion
 * @returns {array} updated state
```

**Review by todofixthis — APPROVED** [review]
LGTM w/ minor suggestions

**Pull request merged**